.storybook/main.ts

```diff
@@ -27,7 +27,13 @@ const config = {
       },
     });
   },
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  stories: [
```

Comment:
This isn't related to this PR but I think comes from some recent storybook-related change. I keep getting this file showing up in my diff when touching PRs related to storybook: `.storybook/main.js`. It looks generated to me, but I am not sure what's generating it. Do you get this too and do you know why it happens?